src/wp-includes/comment.php

```diff
@@ -2425,8 +2425,9 @@ function wp_new_comment_notify_moderator( $comment_id ) {
  */
 function wp_new_comment_notify_postauthor( $comment_id ) {
 	$comment = get_comment( $comment_id );
+	$isNote  = ( $comment && 'note' === $comment->comment_type );
```

Comment:
Naming standards, will need to be changed throughout.

```suggestion
	$is_note = ( $comment && 'note' === $comment->comment_type );
```

Reply:
@adamsilverstein, has been writing a lot of React code lately 😄

Reply:
True! Fixed.